addons/mass_mailing/models/mass_mailing.py

```diff
@@ -799,40 +791,50 @@ def send_mail(self, cr, uid, ids, context=None):
 
             composer_id = self.pool['mail.compose.message'].create(cr, uid, composer_values, context=comp_ctx)
             self.pool['mail.compose.message'].send_mail(cr, uid, [composer_id], auto_commit=True, context=comp_ctx)
-            self.write(cr, uid, [mailing.id], {'sent_date': fields.datetime.now(), 'state': 'done'}, context=context)
+            self.write(cr, uid, [mailing.id], {'state': 'done'}, context=context)
         return True
 
-    def convert_link(self, cr, uid, ids, context=None):
-        website_links = self.pool['website.links']
+    # List of link's href that will not be converted by the shortener
+    links_backlist = ['/unsubscribe_from_list']
```

Comment:
This is not the place it should be implemented. I think the method convert_link should be in website_links, taking as parameter a values parameter that is given to website_links.create() + a blacklist (and not a backlist :p).